client/me/help/help-contact/index.jsx

```diff
@@ -256,15 +256,36 @@ class HelpContact extends React.Component {
 	};
 
 	submitSupportForumsTopic = ( contactForm ) => {
-		const { subject, message } = contactForm;
-		const { currentUserLocale } = this.props;
+		const { site, subject, message, userDeclaresNoSite, userDeclaredUrl } = contactForm;
+		const { currentUserLocale, translate } = this.props;
 
 		this.setState( { isSubmitting: true } );
 		this.recordCompactSubmit( 'forums' );
 
+		let blogHelpMessage = translate( "I don't have a site linked to my WordPress.com account yet" );
+
+		if ( userDeclaresNoSite ) {
+			blogHelpMessage = translate( "I don't have a site with WordPress.com yet" );
+		}
+
+		if ( site || userDeclaredUrl ) {
+			blogHelpMessage = translate( 'The blog I need help with is %(siteUrl)s.', {
```

Comment:
By the way, I think that "site" is the preferred reference over "blog", but this language is identical to the one used in the Forums already when a thread is made there so I'm just copying that.

Reply:
I would go with site, FWIW